mixins/applyDynamicPriceFilter.ts

```diff
@@ -35,10 +35,18 @@ export default (wrapperKey: string = 'dynamicPrice') => ({
     }
   },
   async created () {
-    this.$set(this[wrapperKey], 'values', [
-      this.minAvailablePrice,
-      this.maxAvailablePrice
-    ])
+    if (this.$route.query.hasOwnProperty('price')) {
```

Comment:
I would just use `this.$route.query.price` in check:
![image](https://user-images.githubusercontent.com/30155292/100201556-7be45880-2f00-11eb-9888-b93f8851dbc9.png)